picard/ui/mainwindow.py

```diff
@@ -792,9 +792,12 @@ def play_file(self):
 
     def open_folder(self):
         files = self.tagger.get_files_from_objects(self.selected_objects)
+        opened = []
```

Comment:
Given that use generally won't select more than a few files - and we are avoiding attempting to open the same directory several times which will take much more CPU, the performance impact of a list will be negligible. 

However this would nonetheless be better code so I will change it. 👍